test/README.md

```diff
@@ -0,0 +1,28 @@
+# Unit/Integration Tests
+
+## Requirements
+
+Since TensorFlowOnSpark (TFoS) is literally an integration of TensorFlow and Spark, these tests assume your environment has:
+- Spark installed at `${SPARK_HOME}`
+- Python installed with tensorflow
+- TFoS installed via `pip install -e .` (for easier coding/iteration)
+
+Note: the tests that use Spark will require a local Spark Standalone cluster (vs. Spark local mode), since TFoS assumes that the executors run in separate processes.  This is true for distributed clusters (Standalone and YARN), but not true for the non-distributed Spark local mode, since the executors are just launched as threads in a single process.
+
+## Instructions
```

Comment:
Can we have a simple shell script for this? You could assume that SPARK_HOME is given.